### ISSUE TITLE
Buildings should have variable sizes

### ISSUE THREAD

**Issue by zudijyr** [zudijyr/Building-Sim-Game#22]
Right now every building is exactly one tile in size, which looks odd since they're the same size as the units.  It should be possible to have larger buildings that span multiple tiles.

**Comment by dusktreader** [comment]
Yeah, agreed. We should have a bounding rectangle for buildings, and luckily the geometry module can support that pretty well.

Eventually, we will probably want buildings to have other polygons besides a rectangle, but that's too much of a headache for now
